src/pages/contacts/styles.css

```diff
@@ -1,19 +1,53 @@
 .contacts {
   border: 1px solid #fff;
   box-shadow: 0 0 5px 1px rgb(0 0 0 / 50%);
-  padding: 39px 0 0;
+  padding: 10px 0 10px;
   text-align: center;
   background: rgb(7 21 147 / 50%);
   width: 835px;
   color: #fff;
-  font: 35px/40px "MyriadProRegular";
+  font: 25px/40px "MyriadProRegular";
   margin: 0 auto 100px;
   letter-spacing: 1px;
   height: 487px;
 }
 
-.contacts mark {
-  background: none;
+.contacts h1 {
+  font-size: 30px;
+  margin: 0 0 20px;
+}
+
+.contacts textarea {
+  width: 450px;
+  height: 200px;
+  resize: none;
+  border-radius: 10px;
+}
+
+.contacts input {
+  font: 24px/24px "MyriadProLight";
+  padding: 5px 35px 5px 30px;
```

Comment:
why there is such a huge left padding?

Reply:
This is extra padding. There are enough 5 pixels on all sides. fixed.